sys/net/wot/core/serialization/serialization.c

```diff
@@ -776,20 +776,18 @@ void _serialize_data_schema_number(wot_td_serialize_receiver_t receiver,
                                    wot_td_number_schema_t *schema, wot_td_ser_slicer_t *slicer){
     DEBUG("WoT Serialization: Serializing number data schema\n");
     char buf[16];
-    bool has_minimum = false;
     if (schema->minimum != NULL) {
-        has_minimum = true;
+        _wot_io_send(receiver, ",", 1, slicer);
```

Comment:
I would think so too. Do you have any tests where the current way fails, or why did you remove it?

Reply:
I removed it, because the function `_serialze_data_schema_number` only gets called in `_serialize_data_schema_subclass` (l. 816) and this function only gets called in `_write_data_schema` where it gets called right after `_serialize_json_type`(l.902). This function on the other hand ends with a serialization of a json type and without writing a comma. If you remove the line the TD would look something like this:
```
...
"type": "number"
"minimum": 0.0
...
```